### PR TITLE
Added: Writing Process > Determine the Goal

### PR DESCRIPTION
**Pull request by joshuacc**
@sethsimonds

### CODE REVIEW

File: manuscript/process.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+8. Revise based on feedback
+9. Publish
+
+In this chapter, we will walk through what each of these steps entail.
+
+## 1. Determine the Goal
+
+### Why am I writing this, anyway?
+
+Ever time you start writing something,
```
Comment (tsenart) [inline manuscript/process.md:29]: Every

**Comment by tsenart** [comment]
LGTM! I'm excited to see this book progressing :-)

File: manuscript/process.md (hunk 1 above)
Comment (joshuacc) [inline manuscript/process.md:29]: @tsenart Nice catch.

**Comment by joshuacc** [comment]
@tsenart Thanks for the prose review and encouragement! :grin:

**Pull request merged**